src/epub/text/off-course.xhtml

```diff
@@ -79,7 +79,7 @@
 			<p>Viljalmar Andersen, Secretary-General of the <abbr epub:type="z3998:initialism">U.N.</abbr>, was to introduce the space emissary. “Can you give me an idea at all of what he is like?” he asked nervously.</p>
 			<p>President McCord was as upset as the Dane. He shrugged in agitation. “I know almost as little as you do.”</p>
 			<p>Sir Alfred Oxford protested, “But my dear chap, you’ve had him for almost two weeks. Certainly in that time⁠—”</p>
-			<p>The President snapped back, “You probably won’t believe this, but he’s been asleep until yesterday. When he first arrived he told us he hadn’t slept for a <i>decal</i>, whatever that is; so we held off our discussion with him until morning. Well⁠—he didn’t awaken in the morning, nor the next. Six days later, fearing something was wrong we woke him.”</p>
+			<p>The President snapped back, “You probably won’t believe this, but he’s been asleep until yesterday. When he first arrived he told us he hadn’t slept for a decal, whatever that is; so we held off our discussion with him until morning. Well⁠—he didn’t awaken in the morning, nor the next. Six days later, fearing something was wrong we woke him.”</p>
```

Comment:
This was probably italicized to emphasize the unknown nature of the word. I would replace `<i>` with quotes.

Reply:
I sent an email. Most of the stories have introduced words. My original decision was to leave the first instance in italics and make the rest roman. If we want to change this one, we will have to go through and change them all.

Reply:
I see. In this case since it's a made up English-like word, but is also an actual English word, I would still use double quotes. Italics gives the impression that it's a non-English language, and it also gives the wrong impression that he's emphasizing the English word `decal` as in `sticker`.

Did you add italics throughout all of the book or just the added stories?

Reply:
I didn't "add" the italics. All the stories (even the originals one) had these made up words already in italics. I just removed them except for the first instances.

Reply:
OK, got it. Can you replace both instances with quotes, which I think will make it clear that it's not the emphasized English word, but rather a unique use as a word in a different language.

Reply:
Done. And I caught a few other <i>s I had missed. Its been a tough week apparently.